core/src/main/res/values/strings.xml

```diff
@@ -265,8 +265,8 @@
   <string name="no_bookmarks">No Bookmarks</string>
   <string name="no_history">No History</string>
   <string name="device_default">Device Default</string>
-  <string name="delete_history">Delete History?</string>
-  <string name="delete_bookmarks">Delete Bookmarks?</string>
+  <string name="delete_history">Are you sure you want to delete history?</string>
+  <string name="delete_bookmarks">Are you sure you want to delete bookmark?</string>
```

Comment:
Since there might be more than one bookmark, you could make this "Are you sure you want to delete all bookmarks?"

Reply:
I think this works fine because usually **all** implies clearing all of the history/bookmarks and we use the same message while deleting some of the bookmarks and clearing the entire list.
We can add use bookmark**s**, history is plural already.